### ISSUE TITLE
Decide if protocol spec should be split up into smaller documents.

### ISSUE THREAD

**Issue by joshhus** [hyperledger-archives/fabric#1868]
## Description

The answer apparently, is not yet "yes" to this question, based on the result of the recent PR to split out the security section as a separate doc: https://github.com/hyperledger/fabric/pull/1676

The attempt of that PR was to begin this as an iterative process, by splitting out the Security doc.

Protocol spec: https://github.com/hyperledger/fabric/blob/master/docs/protocol-spec.md

**Comment by ibmmark** [comment]
I agree with the comment in #1676 to take it to the community and discuss... I would suggest to bring it up on the next whitepaper WG, where it might not be the current focus of the WG, would be good to get thought provocation there. If you aren't in the WG, maybe @nickgaski can lead the conversation or I'd be happy too.
